integration/monitor/all-scenarios.js

```diff
@@ -16,13 +42,30 @@ const workers = fs.readdirSync(filesPath).map((file) => {
   return worker;
 });
 
-cron.schedule("* * * * *", (_) => {
+/*
+ * A task runs every minute on the minute.
+ * Spread out the worker runs over the minute to avoid overwhelming the system.
+ * If we receive a message from a worker, it means that there was a failure.
+ * Capture the exception with Sentry and attach a screenshot to the event.
+ */
+cron.schedule("* 5-23 * * *", (_) => {
```

Comment:
📓 Sneaking this in there to only run the monitor from 6am to midnight. A lot of the scenarios start failing when trains and buses stop running.